lib/themes/dosomething/paraneue_dosomething/templates/taxonomy/taxonomy-term.tpl.php

```diff
@@ -13,71 +13,80 @@
   </header>
 
   <?php if (isset($intro)): ?>
-    <section class="container container--intro">
-      <?php if (isset($intro_title)): ?>
-        <h2 class="inline--alt-color"><?php print $intro_title; ?></h2>
-      <?php endif; ?>
-      <div class="wrapper <?php if (isset($intro_image) || isset($intro_video)): print '-half'; else: print '-narrow'; endif; ?>">
-        <?php print $intro; ?>
-        <?php if (isset($modals)): ?>
-          <?php print $modals; ?>
-        <?php endif; ?>
-      </div>
-      <?php if (isset($intro_image) || isset($intro_video)): ?>
-      <div class="wrapper -half">
-        <?php if (isset($intro_video)): ?>
-          <div class="media-video">
-            <?php print $intro_video; ?>
-          </div>
-        <?php elseif (isset($intro_image)): ?>
-          <?php print $intro_image; ?>
+    <section class="container">
+      <div class="wrapper">
+        <div class="container__body <?php if (isset($intro_image) || isset($intro_video)): print '-half'; else: print '-narrow'; endif; ?>">
+          <?php if (isset($intro_title)): ?>
+            <h2 class="inline--alt-color"><?php print $intro_title; ?></h2>
+          <?php endif; ?>
+
+          <?php print $intro; ?>
+          <?php if (isset($modals)): ?>
+            <?php print $modals; ?>
+          <?php endif; ?>
+        </div>
+        <?php if (isset($intro_image) || isset($intro_video)): ?>
+        <div class="container__body -half">
+          <?php if (isset($intro_video)): ?>
+            <div class="media-video">
```

Comment:
Future update, now with the `media` class maybe this changes to `media -video`? Not sure.